app/models/user.rb

```diff
@@ -4,9 +4,8 @@ class User < ApplicationRecord
   has_one_attached :profile_picture
   validates :full_name, :gender, :country, :city, :username, presence: true
   validates :username, uniqueness: true
+  validates :password, :password_confirmation, presence: true
```

Comment:
Same This change is again here too

Reply:
remove these changes from these branches which are not related to their Acceptance criteria of the ticket

Reply:
ok